specs/security.md

```diff
@@ -0,0 +1,26 @@
+# Requirements for security
+
+> A resource is a container,  or a VM
```

Comment:
We can remove this whole paragraph since it assumes there will be an API on the node, this is not the case anymore.